graphql-cli/src/main/java/io/ballerina/graphql/cmd/GraphqlCmd.java

```diff
@@ -93,22 +101,31 @@ public class GraphqlCmd implements BLauncherCmd {
                     "If this is not provided, generate the SDL schema for each GraphQL service in the source file.")
     private String serviceBasePath;
 
+    @CommandLine.Option(names = {"-m", "--mode"}, description = "Ballerina source [service]")
+    private String mode;
+
+    @CommandLine.Option(names = {"-r", "--use-records-for-objects"},
+            description = "Force the generator to generate records" + " types where ever possible")
```

Comment:
If It's a single-line sentence, no need to use the `+` sign there.
```suggestion
            description = "Force the generator to generate records types where ever possible")
```